### DISCUSSION TITLE
what app can i use for ipad(IOS) client

### DISCUSSION THREAD

**Discussion by tg1989h** [v2fly/v2ray-core#2610] [General]
如果你遇到的问题不是 V2Ray 的 bug，比如你不清楚要如何配置，请使用[Discussion](https://github.com/v2fly/v2ray-core/discussions)进行讨论。

此 Issue 会被立即关闭。

If you are not sure if your question is truely a bug in V2Ray, please discuss it [here](https://github.com/v2fly/v2ray-core/discussions) first.

This issue will be closed immediately.

**Comment by MakeFortuneQuietly** [comment]
Client app comparison for various protocols / platforms

https://github.com/net4people/bbs/issues/195
